tasks/modules.py

```diff
@@ -222,10 +222,8 @@ def dependency_path(self, agent_version):
     "pkg/process/util/api": GoModule("pkg/process/util/api", independent=True, used_by_otel=True),
     "pkg/proto": GoModule("pkg/proto", independent=True, used_by_otel=True),
     "pkg/remoteconfig/state": GoModule("pkg/remoteconfig/state", independent=True, used_by_otel=True),
-    "pkg/security/secl": GoModule("pkg/security/secl", independent=True, legacy_go_mod_version=True),
-    "pkg/security/seclwin": GoModule(
-        "pkg/security/seclwin", independent=True, condition=lambda: False, legacy_go_mod_version=True
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is there code still used on `legacy_go_mod_version`? Wondering if there is code still referencing it, if no other module needs it might be removed.